From oliver@neukum.name  Thu Sep 28 15:25:32 2006
From: Oliver Neukum <oliver@neukum.name>
To: greg@kroah.com, linux-usb-devel@lists.sourceforge.net
Subject: USB: new id for kaweth
Date: Thu, 28 Sep 2006 22:21:19 +0200
Content-Disposition: inline
Message-Id: <200609282221.19790.oliver@neukum.name>

this adds a new id to the kaweth driver.
Please apply.

Signed-Off-By: Oliver Neukum <oliver@neukum.name>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/kaweth.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/net/kaweth.c
+++ gregkh-2.6/drivers/usb/net/kaweth.c
@@ -165,6 +165,7 @@ static struct usb_device_id usb_klsi_tab
 	{ USB_DEVICE(0x1645, 0x0005) }, /* Entrega E45 */
 	{ USB_DEVICE(0x1645, 0x0008) }, /* Entrega USB Ethernet Adapter */
 	{ USB_DEVICE(0x1645, 0x8005) }, /* PortGear Ethernet Adapter */
+	{ USB_DEVICE(0x1668, 0x0323) }, /* Actiontec USB Ethernet */
 	{ USB_DEVICE(0x2001, 0x4000) }, /* D-link DSB-650C */
 	{} /* Null terminator */
 };
